features/openid/registration.feature

```diff
@@ -0,0 +1,21 @@
+Feature: OpenID Registration
+  In order to simplify my login process
+  As an unregistered user with an OpenID Account
+  I want to sign up with my OpenID Account
+
+  Scenario: Sign up attempt
+    Given a new user
+    And an unlinked OpenID account with email 'test@test.com'
+    When the account is used to sign in
+    Then the 'E-Mail' input should be filled with 'test@test.com'
```

Comment:
`should already be filled with` maybe? Should be filled sounds like an action (like 'the user should fill in now')